src/metabase/native_query_analyzer.clj

```diff
@@ -134,6 +135,42 @@
                                                     [:= :f.active true]
                                                     (into [:or] (map table-query tables))]}))))
 
+;; NOTE: In future we would want to replace [[considered-drivers]] and [[macaw-options]] with (a) driver method(s),
+;; but given that the interface with Macaw is still in a state of flux, and how simple the current configuration is,
+;; we defer extending the public interface and define the logic locally instead. Macaw itself is not battle tested
+;; outside this same narrow range of databases in any case.
+
+(def ^:private considered-drivers
+  "Since we are unable to ask basic questions of the driver hierarchy outside of that module, we need to repeat"
+  #{:mysql :sqlserver :h2 :sqlite :postgres :redshift})
+
+(defn- macaw-options
```

Comment:
These options are relevant to `find-replace` as well, but I wanted to get eyes on this "code" (actually the comments) sooner than later.